code/js/chrome/omnibox.js

```diff
@@ -0,0 +1,96 @@
+/* global AlgoliaSearch */
+var algolia = new AlgoliaSearch(
+  "TLCDTR8BIO",
+  "686cce2f5dd3c38130b303e1c842c3e3"
+);
+var repositories = algolia.initIndex("repositories");
+
+// Provide help text to the user.
+// chrome.omnibox.setDefaultSuggestion({
+//   description: "Find GitHub repositories..."
+// });
+
+var isFirefox = false;
+if (typeof browser !== 'undefined') {
+  isFirefox = true;
+}
+
+function formatSuggestion(hit) {
+  var content = hit._highlightResult.description.value;
+  content = content.replace(/<(\/)?em>/g, "<$1match>");
```

Comment:
what about just adding this as highlightPreTag and highlightPostTag in the search settings?

Reply:
That would work, but didn't want to dig into the internals of the very old client haha.
Very good point, I'll take a look next iteration.

Reply:
No because validation on marketplaces is sometimes manual it it could highly increase the time with such a change.

I will do the switch to the lite client as have been doing so in another project yesterday.